recipes/tifffolder/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "tifffolder" %}
+{% set version = "0.1.5" %}
+{% set sha256 = "8c068e6bb3de6df35e1487e3be641bfbb85cd1ae7839aaac82622894b76f1ced" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - tifffile >=0.14.0
+
+test:
+  imports:
+    - tifffolder
+
+about:
+  home: https://github.com/tlambert03/tifffolder
+  license: MIT
+  license_family: MIT
```

Comment:
Please add a `license_file`. If one is not distributed with the source distribution on pypi, please include it manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and consider including it in future releases upstream.